src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -182,27 +182,88 @@ button {
     color: red;
 }
 
-@media screen and (max-width: 1000px) {
+@media screen and (max-width: 1024px) {
     .left-form-column input {
-        width: 400px;
+        width: 500px;
+    }
+
+    form .textarea-wrapper textarea {
+        width: 920px;
     }
 }
 
-@media screen and (max-width: 740px) {
-    form {
-        height: 1000px;
+@media screen and (max-width: 768px) {
+    .left-form-column input {
+        width: 600px;
     }
 
-    .textarea-wrapper {
-        margin-top: 200px;
+    form .textarea-wrapper textarea {
+        width: 600px;
     }
 
-    .left-form-column input {
-        width: 330px;
+    .right-form-column {
+        margin-left: 0;
+    }
+
+    .submit-buttons {
+        margin-right: -90px;
     }
 
     .wrapper {
         display: flex;
         flex-direction: column;
     }
 }
+
+
+@media screen and (max-width: 576px) {
+    .left-form-column input {
+        width: 530px;
+    }
+
+    form .textarea-wrapper textarea {
```

Comment:
Please, join this class name to the previous group, instead of writing property 2 times